### PR TITLE
1311 change label text and set color to green

### LINKED ISSUE #1311: "Buy now" is scary , it should be " for sale "
### Is your feature request related to a problem?

/

### Describe the solution you would like

I would like to see

### Describe alternatives you have considered

it should be " for sale " or " available

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇  _ Do a quick check before the merge. 

### PR type
- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional
- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything works
- [ ] I found edge cases

### What's new?
- [x] PR closes #1311, #1382
- [x] Label previously said "BUY NOW" -> changed to "Buy now" and set text color to `#70b364`

### Had issue bounty label ?
- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation
- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot
- [x] My fix has changed **something** on UI, a screenshot for others, is best to understand changes
screenshot-pr-1382
.

### CODE REVIEW

File: styles/global.scss [hunk 1]
```diff
@@ -69,6 +69,12 @@ body {
         text-transform: uppercase;
         font-size: calc(1rem * 0.75);
     }
+    
+    .switch {
```
[the remark this answers is not in the document]
Comment (kkukelka) [reply styles/global.scss:73]: false alarm?

[the remark this answers is not in the document]
Comment (yangwao) [reply styles/global.scss:73]: yeah we should disable it 😄💦

**Comment by roiLeo** [comment]
Wouldn't be better from an UI aspect to have succes color set on active & keep white text? like this:
Screenshot 2021-12-07 at 10-31-53 nft-gallery-nuxt

**Comment by kkukelka** [comment]
> Wouldn't be better from an UI aspect to have succes color set on active & keep white text? like this: Screenshot 2021-12-07 at 10-31-53 nft-gallery-nuxt

And the text color should only be green when the Switch isn't activated?

**Comment by roiLeo** [comment]
> And the text color should only be green when the Switch isn't activated?

I find it weird to have geen text color near all this pink. Only green switch active color feel okayish to me.

**Comment by kkukelka** [comment]
> > And the text color should only be green when the Switch isn't activated?
> 
> I find it weird to have geen text color near all this pink. Only green switch active color feel okayish to me.

in that case we can close this PR, since it doesn't change anything else besides text color

**Comment by yangwao** [comment]
payout sent https://kusama.subscan.io/extrinsic/0x25125be46146ca56f588a48abb2a85687ae1716ad32528c5aff561abb0cfb7a0

let's continue in #1417

**Pull request merged**